svelte/src/lib/UplotSvelte.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+    import 'uplot/dist/uPlot.min.css';
+    import type uPlot from 'uplot';
+    import {onDestroy, onMount} from "svelte";
+    // import {optionsUpdateState} from "uplot-wrappers-common";
+
+    let uPlot: uPlot;
+    export let options: uPlot.Options;
+    export let data: uPlot.AlignedData;
+    export let target: HTMLDivElement | null = null;
+    export let onDelete: (chart: uPlot) => void = () => {};
+    export let onCreate: (chart: uPlot) => void = () => {};
+    export let resetScales:boolean = true;
+
+    let chart: uPlot | null = null;
```

Comment:
Wait, how does it even work, is it a global variable or something? What will happen if one creates 2 instances of the UplotSvelte component? Will the chart from the second instance override the first one or will it somehow create a member variable for that during the runtime? Sorry if it's a stupid question, just want to double check how it actually works.